<ui version="4.0" >
 <class>InternetSettingsPage</class>
 <widget class="QWidget" name="InternetSettingsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Internet Settings</string>
  </property>
  <property name="statusTip" >
   <string>Choose the proxy type and enter proxy host and port if required.</string>
  </property>
  <widget class="QLineEdit" name="proxyUserName" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>170</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>110</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>Port</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="proxyOff" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>30</y>
     <width>371</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>I have a direct connection to the internet</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="proxyManual" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>90</y>
     <width>291</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Use manual proxy settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>110</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>150</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Username</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="proxyFireFox" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>70</y>
     <width>301</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Use Firefox connection settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>150</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Passsword</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="proxyPort" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="proxyHost" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>130</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="proxyPassword" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>170</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="proxyIE" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>341</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Use Internet Explorer connection settings</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
